adapters/network.js

```diff
@@ -196,7 +196,11 @@ const proxyRequest = async request => {
     case "FORM":
       data = new URLSearchParams();
       Object.keys(payload).forEach(key => {
-        data.append(`${key}`, `${payload[key]}`);
+        let payloadKeyStr = `${payload[key]}`;
+        if (Array.isArray(payload[key])) {
+          payloadKeyStr = JSON.stringify(payload[key]);
+        }
+        data.append(`${key}`, payloadKeyStr);
```

Comment:
String template literal will fail for object type as well.
```suggestion
        let payloadValStr = `${payload[key]}`;
        // covers both JSON and Array types
        if (typeof payload[key] === 'object') {
          payloadValStr = JSON.stringify(payload[key]);
        }
        data.append(`${key}`, payloadValStr);
```

Reply:
I did the same before committing but checked all the destinations which had `FORM` and `XML` from the unit-testcases available in the repository. Didn't find any where in an object is being sent. I'm ok with this, let me know your thoughts

Reply:
@sanpj2292 I think we should do that for future proofing. We still dont have any object in FORM values yet but in future we can have

Reply:
@utsabc @saikumarrs Done with this change. Can you please check and let me know if you feel this is ok ?

Reply:
Can we name the variable better? You can see my previous suggestion for reference.

Reply:
Can you check now and confirm ?